.gitignore

```diff
@@ -102,3 +102,6 @@ venv.bak/
 
 # mypy
 .mypy_cache/
+
+.idea/
+asdf.py
```

Comment:
Why do we need to add this to the gitignore?

Reply:
We can rename that to something more reasonable; that's my file that has a complete mockup application for manually testing the library.